account_cash_discount_base/models/account_invoice.py

```diff
@@ -98,15 +101,24 @@ def _compute_has_discount(self):
                 rec.type == 'in_invoice'
             )
 
+    @api.multi
+    @api.depends('date_invoice')
+    def _compute_discount_base_date(self):
+        discount_base_date = datetime.today()
```

Comment:
@GillesTephaneMeyomesse this is not correct to set this outside the loop, as it changes inside.